molecule/release-cluster/host_vars/pulp-webserver

```diff
@@ -1 +1,4 @@
-__pulp_molecule_verify_skip_inspec_tests: false
+__pulp_molecule_verify_skip_testinfra_tests: false
+ansible_python_interpreter: /usr/bin/python3
```

Comment:
I've been meaning for months to clean this up:
https://github.com/pulp/pulp_installer/issues/1166

Are you sure we need to specify it?

Reply:
No, I'm not. I probably copied it from another host_vars file and thought that it was necessary because it was there.